app/src/main/java/io/apicurio/registry/mt/limits/TenantLimitsConfigurationService.java

```diff
@@ -43,52 +44,65 @@ public class TenantLimitsConfigurationService {
 
     @Inject
     @ConfigProperty(defaultValue = "30000", name = "registry.limits.config.cache.check-period")
+    @Info( category = "limits", description = "Cache check period limit", availableSince = "2.1.0.Final")
     Long limitsCheckPeriod;
 
     //All limits to -1 , which means by default all limits are disabled
 
     @Inject
     @ConfigProperty(defaultValue = "-1", name = "registry.limits.config.max-total-schemas")
+    @Info( category = "limits", description = "Max total schemas", availableSince = "2.1.0.Final")
     Long defaultMaxTotalSchemas;
 
     @Inject
     @ConfigProperty(defaultValue = "-1", name = "registry.limits.config.max-schema-size-bytes")
+    @Info( category = "limits", description = "Max schema size (bytes)", availableSince = "2.2.3.Final")
     Long defaultMaxSchemaSizeBytes;
 
     @Inject
     @ConfigProperty(defaultValue = "-1", name = "registry.limits.config.max-artifacts")
+    @Info( category = "limits", description = "Max artifacts", availableSince = "2.1.0.Final")
     Long defaultMaxArtifacts;
     @Inject
     @ConfigProperty(defaultValue = "-1", name = "registry.limits.config.max-versions-per-artifact")
+    @Info( category = "limits", description = "Max versions per artifacts", availableSince = "2.1.0.Final")
     Long defaultMaxVersionsPerArtifact;
 
     //TODO content size
     @Inject
     @ConfigProperty(defaultValue = "-1", name = "registry.limits.config.max-artifact-properties")
+    @Info( category = "limits", description = "Max artifact properties", availableSince = "2.1.0.Final")
     Long defaultMaxArtifactProperties;
     @Inject
     @ConfigProperty(defaultValue = "-1", name = "registry.limits.config.max-property-key-size")
+    @Info( category = "limits", description = "Max artifact property key size", availableSince = "2.1.0.Final")
     Long defaultMaxPropertyKeyBytesSize;
     @Inject
     @ConfigProperty(defaultValue = "-1", name = "registry.limits.config.max-property-value-size")
+    @Info( category = "limits", description = "Max artifact property value size", availableSince = "2.1.0.Final")
     Long defaultMaxPropertyValueBytesSize;
 
     @Inject
     @ConfigProperty(defaultValue = "-1", name = "registry.limits.config.max-artifact-labels")
+    @Info( category = "limits", description = "Max artifact labels", availableSince = "2.2.3.Final")
     Long defaultMaxArtifactLabels;
     @Inject
     @ConfigProperty(defaultValue = "-1", name = "registry.limits.config.max-label-size")
+    @Info( category = "limits", description = "Max artifact max label size", availableSince = "2.1.0.Final")
     Long defaultMaxLabelBytesSize;
```

Comment:
Should this be `Max artifact label size`? (same applies to next 3 props)

Reply:
👍